src/Settings.cpp

```diff
@@ -134,6 +134,12 @@ namespace Settings
 			});
 
 		gchar** pinnedListBuffer = g_key_file_get_string_list(mFile, "user", "pinned", NULL, NULL);
+		
+		dockSize.setup(g_key_file_get_integer(mFile, "user", "dockSize", NULL),
```

Comment:
Move this block down beneath the pinnedAppList, so it's not in between two related pieces of code.